proof/invariant-abstract/SchedContext_AI.thy

```diff
@@ -80,26 +80,29 @@ lemma get_refills_wp:
   \<lbrace> \<lambda>r s. P r s\<rbrace>"
   by (wpsimp simp: get_sched_context_def get_refills_def wp: get_object_wp) fastforce
 
-lemma refills_merge_valid:
-  "ls \<noteq> [] \<Longrightarrow> refills_merge_prefix ls \<noteq> []"
-  by (induct ls rule: refills_merge_prefix.induct; simp)
+lemmas refill_unblock_check_defs
+  = refill_unblock_check_def is_round_robin_def merge_refills_def refill_pop_head_def
+    refill_head_overlapping_loop_def set_refill_hd_def update_refill_hd_def
```

Comment:
Nice! It can be a pain to find them sometimes, but I always like definition sets like these.